### PR TITLE
[bug 1159515] Disallow new signups and banners.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@bensternthal messaging r?

**Comment by bensternthal** [comment]
Got it, i'll have to get official messaging. Will update here when we have it.

**Comment by bensternthal** [comment]
Testing locally I am unable to sign in at all. I get the browser id prompt, submit and nothing happens. Refreshing page confirms I am not logged in. Tested ff and chrome. No errors.

**Comment by glogiotatidis** [comment]
Are you able to login when on master branch? Opening up 'network' tab on developer tools will give show you the browserid request, I suspect that it returns a 500?

**Comment by bensternthal** [comment]
Yep 500 error on master.

**Comment by glogiotatidis** [comment]
OK so it's not my changes. Select the 500 request under the network tab in developer tools again and click the 'response' tab. This will give us a meaningful error hopefully.

**Comment by bensternthal** [comment]
Heh, easier said then done. I had to prevent the browser ID window from closing so I could see the request.

That being said, attached is the output. Note its a 200 from the browser id pop up, but my console shows a 500 post request.


screen shot 2015-06-02 at 1 46 49 pm

**Comment by bensternthal** [comment]
Note copy is still being discussed, once final I will let you know with a link to the doc.

**Comment by glogiotatidis** [comment]
> Heh, easier said then done. I had to prevent the browser ID window from closing so I could see the request.

you don't need the console from the browserid window but instead from the original window. that's why you don't see the 500 request.

**Comment by glogiotatidis** [comment]
> Note copy is still being discussed, once final I will let you know with a link to the doc.

ping for the copy @bensternthal. Ideally we should get this ready by the end of this week before I go to conferences or PTO.

**Comment by bensternthal** [comment]
Noted, I have been pinging in the bug for updates. Just emailed and will nag in the project status meeting today.

**Comment by bensternthal** [comment]
@glogiotatidis I do not think you are blocked on this one anymore -- If we can get this merged I am happy to test and we can push live this week.

**Comment by glogiotatidis** [comment]
@akatsoulas r?

File: affiliates/base/views.py [hunk 1]
```diff
@@ -93,3 +96,11 @@ def handler500(request):
 
 def strings(request):
     return render(request, 'base/strings.html')
+
+
+class BrowserIDVerify(Verify):
+    def login_failure(self, msg=None):
```
Comment (akatsoulas) [inline affiliates/base/views.py]: I would go with `''`

File: affiliates/base/templates/base/dashboard.html [hunk 2]
```diff
@@ -29,42 +29,20 @@ <h1 class="page-title">{{ _('Dashboard') }}</h1>
...
+        <div class="contain">
+          <header>
+            <h2 class="section-title">{{ _('My Banners') }}</h2>
+          </header>
+            <ul class="banners-list">
+              {% for link in links %}
+                {{ link.preview_html(link.get_absolute_url()) }}
+              {% endfor %}
+            </ul>
+        </div>
```
Comment (akatsoulas) [inline affiliates/base/templates/base/dashboard.html]: The indentation seems off here.

**Comment by akatsoulas** [comment]
After a quick test everything seems to be working file. I don't know if you are interested in adding a few tests for the new functionality. r+wc :cocktail:

File: affiliates/base/views.py (hunk 1 above)
Comment (glogiotatidis) [inline affiliates/base/views.py]: +1

File: affiliates/base/templates/base/dashboard.html (hunk 2 above)
Comment (glogiotatidis) [inline affiliates/base/templates/base/dashboard.html]: nice catch

**Comment by glogiotatidis** [comment]
Thanks @akatsoulas! :ice_cream:

**Pull request merged**